src/components/SearchBox/SearchBox.tsx

```diff
@@ -35,7 +35,7 @@ export type Props = PropsWithSpread<
     /**
      * A function that is called when the user clicks the search icon or presses enter
      */
-    onSearch?: () => void;
+    onSearch?: (inputValue: string) => void;
```

Comment:
I think this might be a breaking change.

Reply:
This is OK as it is backwards compatible. Existing `onSearch` functions like `() => "something"` or with a type of `() => void` will continue to work.